Ethinylestradiol/Ethinylestradiol-Report.md

```diff
@@ -0,0 +1,419 @@
+# 5 References
+**Back 1979** Back DJ, Breckenridge AM, Crawford FE, et al. An investigation of the pharmacokinetics of ethynylestradiol in women using radioimmunoassay. *Contraception*. 1979;20(3):263-273.
+
+**Back 1981** Back DJ, Bates M, Breckenridge AM, et al. The pharmacokinetics of levonorgestrel and ethynylestradiol in women - studies with Ovran and Ovranette. *Contraception*. 1981;23(3):229-239.
+
+**Back 1987** Back DJ, Grimmer SF, Rogers S, Stevenson PJ, Orme ML. Comparative pharmacokinetics of levonorgestrel and ethinyloestradiol following intravenous, oral and vaginal administration. *Contraception*. 1987;36(4):471-479.
+
+**Balogh 1995** Balogh A, Boerner A, Kuhnz W, Klinger G, Vollanth R, Henschel L. Influence of ethinylestradiol-containing combination oral contraceptives with gestodene or levonorgestrel on caffeine elimination. *Eur J Clin Pharmacol*. 1995;48(2):161-166.
+
+**Chang 2009** Chang, S. Y., Chen, C., Yang, Z., Rodrigues, A. D. (2009). Further assessment of 17α-ethinyl estradiol as an inhibitor of different human cytochrome P450 forms in vitro. *Drug Metabolism and Disposition*, 37(8), 1667-1675.
+
+**DrugBank DB00977** (https://www.drugbank.ca/drugs/DB00977)
+
+**Ezuruike 2018** Ezuruike U, Humphries H, Dickins M, Neuhoff S, Gardner I, Rowland Yeo K. Risk–Benefit Assessment of Ethinylestradiol Using a Physiologically Based Pharmacokinetic Modeling Approach. *Clin Pharmacol Ther*. 2018;104(6):1229-1239
+
+**FDA. QUARTETTE ** FDA. QUARTETTE (levonorgestrel/ethinyl estradiol and ethinyl estradiol) tablets, for oral use. Website: accessdata.fda.gov/drugsatfda_docs/label/2013/204061s000lbl.pdf.2013.
+
+**Goebelsmann 1986** Goebelsmann U, Hoffman D, Chiang S, Woutersz T. The relative bioavailability of levonorgestrel and ethinyl estradiol administered as a low-dose combination oral contraceptive. *Contraception*. 1986;34(4):341-351.
+
+**Granfors 2005** Granfors MT, Backman JT, Laitila J, Neuvonen PJ. Oral contraceptives containing ethinyl estradiol and gestodene markedly increase plasma concentrations and effects of tizanidine by inhibiting cytochrome P450 1A2. *Clin Pharmacol Ther*. 2005;78(4):400-411.
+
+**Karjalainen 2008** Karjalainen, M. (Thesis, 2008). Inhibition of CYP1A2-mediated drug metabolism in vitro and in humans: With special emphasis on rofecoxib and other NSAIDs. Website: helda.helsinki.fi/bitstream/handle/10138/23039/inhibiti.pdf?sequence=1&origin=publication_detail
+
+**Kothare 2012** Kothare PA, Seger ME, Northrup J, Mace K, Mitchell MI, Linnebjerg H. Effect of exenatide on the pharmacokinetics of a combination oral contraceptive in healthy women: an open-label, randomised, crossover trial. *BMC Clin Pharmacol*. 2012;12:8. 
```

Comment:
randomi**z**ed

Reply:
No, this is how the publication is titled (british english ;) )

Reply:
ok...